.github/workflows/release.yml

```diff
@@ -134,7 +134,8 @@ jobs:
             exit 1
```

Comment:
The code changes made in this review include updating the code to store the sanitized JIRA issue key in a temporary file named temp.env and then appending the contents of this file to the $GITHUB_ENV file. This change ensures that the sanitized JIRA issue key is stored correctly and can be accessed by subsequent steps in the workflow.